app/View/Login.php

```diff
@@ -10,32 +10,53 @@
  .error { 
   }
 </style>
+<script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js">
+</script>
+<script type="text/javascript" src="http://code.jquery.com/ui/1.8.18/jquery-ui.min.js"></script>
+
+<script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
```

Comment:
All these versions are different. Two versions of jquery are used. All these files should be in our project so we can control their versions.